src/Graph.RBAC/Graph.RBAC/Microsoft.Azure.Graph.RBAC.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Graph.RBAC
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Graph.RBAC">
-      <PackageVersion>1.8.0-preview</PackageVersion>
+      <PackageVersion>2.0.0-preview</PackageVersion>
```

Comment:
I was told to increase the major version since it contains breaking changes. But as 2.0.0 is used by autorest, and with offline conversation with Hovsep, i moved the version to 1.9.0-preview.